<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rate_Calculator</class>
 <widget class="QMainWindow" name="Rate_Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lodging Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>157</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="rate_label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Rate</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="aaa_rate_edit">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="soe_rate_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="aaa_label">
       <property name="text">
        <string>AAA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rack_label">
       <property name="text">
        <string>RACK</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="rack_rate_edit"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lbms_rate_edit"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="rack_total_edit"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="aaa_total_edit"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lbms_total_edit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="soe_label">
       <property name="text">
        <string>SOE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbms_label">
       <property name="text">
        <string>LBMS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="soe_total_edit"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="total_label">
       <property name="text">
        <string>Total</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculate_button">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="configure_menu">
    <property name="title">
     <string>&amp;Configure</string>
    </property>
    <addaction name="constants_action"/>
   </widget>
   <addaction name="configure_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="constants_action">
   <property name="text">
    <string>&amp;Constants</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>rack_rate_edit</tabstop>
  <tabstop>rack_total_edit</tabstop>
  <tabstop>aaa_rate_edit</tabstop>
  <tabstop>aaa_total_edit</tabstop>
  <tabstop>lbms_rate_edit</tabstop>
  <tabstop>lbms_total_edit</tabstop>
  <tabstop>soe_rate_edit</tabstop>
  <tabstop>soe_total_edit</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>calculate_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>rack_rate_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>rack_total_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>aaa_rate_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>aaa_total_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>lbms_rate_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>lbms_total_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>soe_rate_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>soe_total_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
